metadata/site_import/meta/system-objecttype-extensions.xml

```diff
@@ -264,6 +264,13 @@
 
     <type-extension type-id="SitePreferences">
         <custom-attribute-definitions>
+            <attribute-definition attribute-id="Adyen3DS2Enabled">
```

Comment:
If you set this, is it going to be shown on the admin of the platform? Can we hide it?

Reply:
showing it is fine as they should be able to turn it on.

Reply:
It won't be shown in the business manager. This just creates the variable, but not displayed in custom preferences.